### ISSUE TITLE
Moving the reference - IATI Standard

### ISSUE THREAD

**Issue by benjamincoleman** [IATI/IATI-Standard-Website#392]
This is part of the "moving the reference" task that deals with "the standard" content. 

You can see this new section in the IA below: 

![Screen Shot 2019-12-09 at 16 05 25](https://user-images.githubusercontent.com/64877/70451800-f0c74400-1a9d-11ea-878f-c1f5ea8c9c99.png)

**The Standard**
The top level page uses this template: http://iati.surge.sh/page-templates/09-the-standard and this links into various pages underneath. 

**v2.03**
This is a version of the reference. There are multiple versions as you can see in the IA. 

The structure and content here comes from the outputted JSON when you run the SSOT. Ask Alex about that if you need to know more. 

Each of the pages for any of the versions of the standard uses this template: http://iati.surge.sh/page-templates/09-1-reference and these variations: 

- http://iati.surge.sh/page-templates/09-1a-codelist
- http://iati.surge.sh/page-templates/09-1b-schema
- http://iati.surge.sh/page-templates/09-1c-organisation-identifiers

**Upgrade Changelog**

I believe this is also controlled by the SSOT, but we'll need to check with Alex + John on that. 

**How we manage the standard**
This page, and its children use the T1.1 template which we already use on the website for other areas like the about section: http://iati.surge.sh/page-templates/01-1-about-subpage. It will be content managed by Wagtail, and edited by the BAs Amy and Petya.

**Comment by benjamincoleman** [comment]
@PetyaKangalova Questions I have on this one are: 

1. Do you know whether the "upgrade changelog" branch of this section will be content managed in the SSOT, or in Wagtail? 

2. Am I right in thinking the "How we manage the standard" section should be content managed in Wagtail?

**Comment by PetyaKangalova** [comment]
@benjamincoleman 

> Do you know whether the "upgrade changelog" branch of this section will be content managed in the SSOT, or in Wagtail?

The upgrade change log links to all pages in here: http://reference.iatistandard.org/203/upgrades/all-versions/ I think this should me managed in SSOT - it is not part of the SSOT repo per se, but refers to changes in the SSOT so I would not move the content to Wagtail. It will be good though to get a dev view on this- @Ocre42  or @akmiller01 ? 

> Am I right in thinking the "How we manage the standard" section should be content managed in Wagtail?

This would include moving the content for Upgrade process (http://reference.iatistandard.org/203/upgrades/) and Codelist Management (http://reference.iatistandard.org/203/codelists-guides/codelist-management/) which will be managed on Wagtail.

**Comment by Ocre42** [comment]
@benjamincoleman 

> Do you know whether the "upgrade changelog" branch of this section will be content managed in the SSOT, or in Wagtail?
These pages are currently handled by the [guidance section of the SSOT](https://github.com/IATI/IATI-Guidance), which is brought in as an external part of the current generation process. I would be tempted to move these to the wagtail site as that repo isn't updated very often and a lot of the content will be moving to Wagtail regardless. It might be worth having a brief conversation about this when we're back in January @akmiller01 @PetyaKangalova

**Comment by PetyaKangalova** [comment]
@Ocre42  thanks for the quick response! I was actually struggling to find where the change log fits. Yes, you have a point as new change logs will happen only after upgrades and we can also set up a more automated way of recording the changes.

I will set a meeting for the first week of January with the team, so that we can agree and confirm for all different sections of the website where the content fits. @benjamincoleman and @k8hughes , does that sounds okay in terms of timeline?

**Comment by PetyaKangalova** [comment]
@benjamincoleman we just discussed this with the team and agreed that the  **Upgrade Changelog** and **How we manage the standard** will be managed on Github as they are now- **not** moving any of the content management to Wagtail.

**Comment by benjamincoleman** [comment]
@PetyaKangalova ok thanks
